<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TipManager</class>
 <widget class="QWidget" name="TipManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>558</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TipManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(94,96,98);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="RightLayout" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="rightLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_memo">
          <property name="styleSheet">
           <string notr="true">color: rgb(255,255,255);</string>
          </property>
          <property name="text">
           <string>Memo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="memoTextEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit:enabled {
color: rgb(0,0,0);
background-color: rgb(240,255,255);
border-radius:4px;
}

QTextEdit:disabled {
color: rgb(155,155,155);
background-color: rgb(240,255,255);
border-radius:4px;
}
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_file">
            <property name="styleSheet">
             <string notr="true">color: rgb(255,255,255);</string>
            </property>
            <property name="text">
             <string>File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dirEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240,255,255);
border-radius: 2px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(240,255,255);
border-radius:4px;</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="newButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
QPushButton:enabled {
color: rgb(255,255,255);
}

QPushButton:disabled {
border-style: hidden;
color: rgb(155,155,155);
}

QPushButton{
width: 90px;
height: 25px;
}</string>
            </property>
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:enabled {
color: rgb(255,255,255);
}

QPushButton:disabled {
border-style: hidden;
color: rgb(155,155,155);
}

QPushButton{
width: 90px;
height: 25px;
}</string>
            </property>
            <property name="text">
             <string>Edit</string>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:enabled {
color: rgb(255,255,255);
}

QPushButton:disabled {
border-style: hidden;
color: rgb(155,155,155);
}

QPushButton{
width: 90px;
height: 25px;
}</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="loadButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:enabled {
color: rgb(255,255,255);
}

QPushButton:disabled {
border-style: hidden;
color: rgb(155,155,155);
}

QPushButton{
width: 90px;
height: 25px;
}</string>
            </property>
            <property name="text">
             <string>Load Dir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:enabled {
color: rgb(255,255,255);
}

QPushButton:disabled {
border-style: hidden;
color: rgb(155,155,155);
}

QPushButton{
width: 90px;
height: 25px;
}</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="aboutButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:enabled {
color: rgb(255,255,255);
}

QPushButton:disabled {
border-style: hidden;
color: rgb(155,155,155);
}

QPushButton{
width: 90px;
height: 25px;
}</string>
            </property>
            <property name="text">
             <string>About</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="LeftLayout" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>480</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="leftLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255,255,255);</string>
          </property>
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="titleEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit:enabled {
color: rgb(0,0,0);
background-color: rgb(240,255,255);
border-radius:2px;
}

QLineEdit:disabled {
color: rgb(155,155,155);
background-color: rgb(240,255,255);
border-radius:2px;
}
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_code">
          <property name="styleSheet">
           <string notr="true">color: rgb(255,255,255);</string>
          </property>
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="codeTextEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit:enabled {
color: rgb(0,0,0);
background-color: rgb(240,255,255);
border-radius:4px;
}

QTextEdit:disabled {
color: rgb(155,155,155);
background-color: rgb(240,255,255);
border-radius:4px;
}
</string>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
